README.md

```diff
@@ -28,42 +41,42 @@ There are several ways to install this app onto a workload cluster.
 
 ```yaml
 # values.yaml
-
+global:
+  remoteWrite:
+  - name: "remotewrite" 
+    url: "http://$BASEDOMAIN/write"
+
+prometheus-agent:
+  serviceMonitor:
+    enabled: true
+    relabelings:
+    # Add app label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_app_kubernetes_io_name
+      targetLabel: app
+    # Add instance label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_app_kubernetes_io_instance
+      targetLabel: instance
+    # Add node label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_node_name
+      targetLabel: node
+    # Add team label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_application_giantswarm_io_team
+      targetLabel: team
 ```
 
-### Sample App CR and ConfigMap for the management cluster
+### Conflicts 
 
-If you have access to the Kubernetes API on the management cluster, you could create
-the App CR and ConfigMap directly.
-
-Here is an example that would install the app to
-workload cluster `abc12`:
+To avoid conflicts with other prometheuses deployed to same cluster,
+you can use selectors for `ServiceMonitors` and `PodMonitors`:
 
 ```yaml
-# appCR.yaml
-
+prometheus-agent:
+  serviceMonitorSelector:
+    cluster: "c8dfg"
```

Comment:
I don't think selecting by cluster name would solve conflicts between giantswarm and customer's servicemonitors.
And if it does, it's not very intuitive.
I would rather push for something like `organization: giantswarm` vs `organization: lovelycustomer`.

Reply:
I'm not really sure that we have a label that differenciate our apps vs theirs. That is probably something we should RFC

Reply:
Asking internally: https://gigantic.slack.com/archives/CLPMFRVU6/p1667820439946089

Reply:
I'm not sure about `organization: giantswarm` 
but that's fine.